### PR TITLE
Add Typing Sound Feature

### PR DESCRIPTION
**Pull request by masabinhok**
This PR introduces a typing sound feature to enhance user experience. The changes allow users to toggle the typing sound on or off and currently include a test sound, which can be changed later. #119

Features
Typing Sound: Added audio feedback when the user types.
Sound Toggle: Users can toggle the sound on or off.
Input Filtering: The typing sound ignores key presses from modifiers such as Ctrl, Tab, Shift, and Caps Lock.
Test Sound: Implemented a temporary sound for testing purposes, which can be replaced in future iterations.

Video Preview: 
https://github.com/user-attachments/assets/59e97f35-410d-4695-8b50-94ee4ed871b5

Note
A few files were modified to implement this feature. Due to Husky pre-commit rules, the number of files changed may appear higher than expected. Please don't be overwhelmed by the changes.

### CODE REVIEW

**Comment by JhohannesK** [comment]
Great.
I'll take a look at this later, please.

**Comment by masabinhok** [comment]
Yeah, sure take your time.

**Review by JhohannesK — COMMENTED** [review]
Can you exempt the .github folder and content from being pushed.

Add it to .prettierignore

File: package.json
Comment (JhohannesK) [inline package.json:1]: You never requested and explained why you were editing the package.json as instructed in the contributing.md file.

File: components/Header.tsx [hunk 1]
```diff
@@ -11,14 +12,15 @@ function Header() {
                TipoHiz.__
             </p>
          </div>
-         <div className='flex justify-between items-center gap-3 font-semibold'>
+         <div className="flex justify-between items-center gap-3 font-semibold">
             <div className="flex items-center justify-center p-2 rounded-xl gap-3 cursor-pointer text-foreground bg-input">
                <p className="text-sm tracking-wide text-background">Login</p>
-               <LuUser className='text-background' />
+               <LuUser className="text-background" />
```
Comment (JhohannesK) [inline components/Header.tsx:18]: I don't neither your husky nor prettier works.

We use single quotes. check .prettierrc

File: app/(Base)/page.tsx [hunk 2]
```diff
@@ -28,20 +40,27 @@ export default function HomePage() {
                >
                   start
                </button> */}
-               <div className='mb-2'>{timer}</div>
+               <div className="mb-2">{timer}</div>
             </div>
             <div className="flex items-center justify-center mb-4 tracking-widest lowercase text-input gap-x-1">
-               <MdLanguage className='mt-2'/>
+               <MdLanguage className="mt-2" />
```
Comment (JhohannesK) [inline app/(Base)/page.tsx:46]: single quotes

File: .github/update-contributors.yml
Comment (JhohannesK) [inline .github/update-contributors.yml:1]: revert commit of this file

File: .github/ISSUE_TEMPLATE/feature_request.yml
Comment (JhohannesK) [inline .github/ISSUE_TEMPLATE/feature_request.yml:1]: revert commit of this file

File: .github/ISSUE_TEMPLATE/bug_report.yml
Comment (JhohannesK) [inline .github/ISSUE_TEMPLATE/bug_report.yml:1]: revert commit of this file

**Review by JhohannesK — CHANGES_REQUESTED** [review]

**Comment by JhohannesK** [comment]
Please join the discord if you can and ask any follow up questions you may have.
https://discord.gg/8Z67Fv8z

**Comment by JhohannesK** [comment]
@masabinhok is there anything I need to help with?!

**Comment by masabinhok** [comment]
yeah i couldnot revert the github files. Can you explain me why do i need to revert them please?

**Comment by JhohannesK** [comment]
> yeah i couldnot revert the github files. Can you explain me why do i need to revert them please?

Your prettier formatted those files.

**Comment by masabinhok** [comment]
These issues are fixed in the newer commits; I have the same .github folder as your preview branch. 
Also, you can see the changes: previously double quote changed to single quote because of prettierrc and husky i think. 
image

But, now its back to normal. The project runs smoothly, and i think this pr is good to go?

**Pull request merged**